### ISSUE TITLE
"Character Counter" Missing

### ISSUE THREAD

**Issue by corsair** [sparksuite/simplemde-markdown-editor#603]
**Description**
While setting up SimpleMDE, it appears that while a character counter is mentioned (and a style class is added for it in 1.10.0 / 6318fc1b193fc419a0aede13738b2be07355b16a), it was never actually implemented into the editor.

Attempting to add "characters" to status's array properly adds "characters: " to the status bar, however does not actually have any functionality associated with it, thus does not display numbers nor increase/decrease with input.

**Example:**

![image](https://user-images.githubusercontent.com/1847891/27888523-457cf044-619b-11e7-9d44-48c6bac980ec.png)

**Setup:**
```
var simplemde = new SimpleMDE({
  autosave: "enabled",
  toolbar: ["bold", "italic", "|", "link", "|", "preview"],
  status: ["characters"],
});
```

**Comment by Nrde** [comment]
Yes, the actual code seems to be missing. Fortunately it is relatively easy to add it yourself.
```
status: ["autosave", "lines", "words", "cursor", {
		className: "characters",
		defaultValue: function(el) {
			el.innerHTML = "0";
		},
		onUpdate: function(el) {
			el.innerHTML = simplemde.value().length;
		}
	}]

**Comment by mobeendude** [comment]
Not able to get reference to simplemde variable in this code ? Anything, I am missing?

**Comment by davuses** [comment]
> Not able to get reference to simplemde variable in this code ? Anything, I am missing?

@mobeendude
I actually have to select the div element to make this work:
```
 onUpdate: function (el) {
                el.innerHTML = document.querySelector(
                  "div.CodeMirror.cm-s-paper.CodeMirror-wrap").innerText.length;
              }
```
